### PR TITLE
fixes #8460 - adds font-awesome support

### PR DESCRIPTION
**Pull request by ohadlevy**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -23,6 +23,7 @@ gem 'foreigner', '~> 1.4.2'
 gem 'validates_lengths_from_database',  '~> 0.2.0'
 gem 'friendly_id', '~> 4.0'
 gem 'secure_headers', '~> 1.3.3'
+gem 'font-awesome-rails' # Added here and not in asset group as the engine contains a few helpers
```
Comment (domcleal) [inline Gemfile:26]: The gem author says it follows semver, it might be best to pin to a major version at least.

**Comment by domcleal** [comment]
What does it actually do?  I take it you have to actually use it, it doesn't change anything?

**Comment by ohadlevy** [comment]
this just enables the assets, in order to use the icons from:
http://fortawesome.github.io/Font-Awesome/icons/

**Comment by domcleal** [comment]
Yeah, I'm not sure I see the point in a new dep until it's actually used.

**Comment by ohadlevy** [comment]
OK, do not merge yet then.

**Comment by dLobatog** [comment]
@ohadlevy One good excuse to use this is to replace our spinners by Font Awesome spinners with the fa-spin class, they look better in any size :smile:

**Comment by ehelms** [comment]
We use and provide font-awesome (alongside Patternfly -- currently upgrading to latest https://github.com/Katello/bastion/pull/10) in the Bastion gem and migrating to it's use would provide more consistency between us.

**Comment by ohadlevy** [comment]
@ehelms not sure that patternfly is not an overkill for this pr, does it make sense to move more common ux to core from katello/bastion instead?

**Comment by ehelms** [comment]
Definitely, unless we want to a move to a model of having UI/assets kept in
a UI focused gem. I have been thinking that similar to Rubocop, adding font
awesome and paternfly and transitioning pieces in smaller PRs might be
easier than doing all at once.

**Comment by dLobatog** [comment]
@ohadlevy Mind if we close this, or maybe update some icons (spinners for instance as I mentioned) to justify keeping it open? Also the Patternfly PR includes this as well

**Pull request closed without merging**